### PR TITLE
[WORKFLOWS/ETHEREUM-CONTRACTS] reduce some gda pr noise

### PR DESCRIPTION
**Pull request by 0xdavinchee**
## Summary

I have taken some stuff that I've done in the GDA branch and done it here to remove some of the noise in that PR:
- lcov filtering in ci.feature
- semantic money remappings for automation-contracts
- create SuperfluidFrameworkDeploymentSteps and utilize

### CODE REVIEW

File: packages/ethereum-contracts/dev-scripts/deploy-test-framework.js [hunk 1]
```diff
@@ -141,9 +170,20 @@ const deployTestFramework = async () => {
...
+                CFAv1ForwarderDeployerLibrary:
+                    CFAv1ForwarderDeployerLibrary.address,
+                SuperfluidLoaderDeployerLibrary:
+                    SuperfluidLoaderDeployerLibrary.address,
             },
         }
     );
+    const numSteps = await frameworkDeployer.getNumSteps();
+    for (let i = 0; i < numSteps; i++) {
+        await frameworkDeployer.executeStep(i);
```
Comment (hellwolf) [inline packages/ethereum-contracts/dev-scripts/deploy-test-framework.js]: I couldn't find where this executeStep is defined :D ??

Comment (0xdavinchee) [reply packages/ethereum-contracts/dev-scripts/deploy-test-framework.js]: it's in `SuperfluidFrameworkDeploymentSteps`

File: packages/automation-contracts/scheduler/test/SuperfluidTester.sol [hunk 2]
```diff
@@ -4,7 +4,13 @@ pragma solidity ^0.8.0;
 import { Test } from "forge-std/Test.sol";
 import { IERC1820Registry } from "@openzeppelin/contracts/utils/introspection/IERC1820Registry.sol";
 import { ERC1820RegistryCompiled } from "@superfluid-finance/ethereum-contracts/contracts/libs/ERC1820RegistryCompiled.sol";
-import { Superfluid, ConstantFlowAgreementV1, InstantDistributionAgreementV1, SuperTokenFactory, SuperfluidFrameworkDeployer } from "@superfluid-finance/ethereum-contracts/contracts/utils/SuperfluidFrameworkDeployer.sol";
+import { SuperfluidFrameworkDeployer } from "@superfluid-finance/ethereum-contracts/contracts/utils/SuperfluidFrameworkDeployer.sol";
```
Comment (hellwolf) [inline packages/automation-contracts/scheduler/test/SuperfluidTester.sol:7]: tech debt: to reuse the ethereum-contracts's FoundrySuperfluidTester.sol here

@elvijsTDL

**Review by hellwolf — APPROVED** [review]
steppped deployers ftw

**Pull request merged**